src/lib/button/button.ts

```diff
@@ -110,6 +110,8 @@ export class MdButton {
     return el.hasAttribute('md-' + suffix) || el.hasAttribute('mat-' + suffix);
   });
 
+  _isIconButton: boolean = this._getHostElement().hasAttribute('md-icon-button');
```

Comment:
The `: boolean` part of this should be unnecessary since TypeScript infers the type from the right hand side.